cmd/kyma/sync/function/opts.go

```diff
@@ -0,0 +1,52 @@
+package function
+
+import (
+	"fmt"
+	"github.com/kyma-project/cli/internal/cli"
+	"os"
+	"time"
+)
+
+//Options defines available options for the command
+type Options struct {
+	*cli.Options
+
+	Name       string
+	Namespace  string
+	OutputPath string
+	Timeout    time.Duration
+}
+
+//NewOptions creates options with default values
+func NewOptions(o *cli.Options) *Options {
+	options := &Options{Options: o}
+	return options
+}
+
+func (o *Options) setDefaults(defaultNamespace string) (err error) {
+	if o.OutputPath == "" {
+		o.OutputPath, err = os.Getwd()
+		if err != nil {
+			return err
+		}
+	}
+
+	setIfZero(&o.Namespace, defaultNamespace, "default")
```

Comment:
This is not necessary, `c.K8s.DefaultNamespace()` already returns `default` if the user has no default namespace configured in the Kubeconfig.
Check the documentation please:
https://github.com/clebs/kyma-cli/blob/bad193c82b5fa538f69335789e64bb536beb2200/internal/kube/kube.go#L32-L35